src/systemTest/java/com/flightstats/hub/model/WebhookCallbackSetting.java

```diff
@@ -0,0 +1,17 @@
+package com.flightstats.hub.model;
+
+import lombok.Builder;
+
+import java.beans.ConstructorProperties;
+
+@Builder
+public class WebhookCallbackSetting {
+    Integer triesUntilSuccess;
```

Comment:
tryUntilSuccees?

Reply:
This matches what's in the callback server. It is the number of tries it'll make, so I think plural makes sense. I'd go for `try` singular if it were a boolean, though.